dizoo/mujoco/config/mbrl/halfcheetah_mbsac_mbpo_config.py

```diff
@@ -0,0 +1,115 @@
+from easydict import EasyDict
```

Comment:
why remove humanoid config

Reply:
Because in the mujoco directory, I do not find any other algorithms with `humanoid` config. So I just delete this for consistency.